modules/menu/css/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@workday/canvas-kit-css-menu",
+  "version": "3.0.0-alpha.0",
+  "description": "Menu CSS for Canvas kit CSS",
+  "author": "Workday, Inc. (https://www.workday.com)",
+  "license": "Apache-2.0",
+  "files": [
+    "dist",
+    "lib",
+    "index.scss"
+  ],
+  "style": "dist/canvas-kit-css-menu.css",
```

Comment:
Nit: Can you reference the .min files (`dist/canvas-kit-css-menu.min.css`) instead? Same for the other packages